pkg/storage/replica_raftstorage.go

```diff
@@ -806,172 +800,144 @@ func (r *Replica) applySnapshot(
 	}
 
 	var stats struct {
-		clear   time.Time
-		batch   time.Time
-		entries time.Time
-		commit  time.Time
-	}
-
-	var size int
-	for _, b := range inSnap.Batches {
-		size += len(b)
-	}
-	for _, e := range inSnap.LogEntries {
-		size += len(e)
-	}
-
-	log.Infof(ctx, "applying %s snapshot at index %d "+
-		"(id=%s, encoded size=%d, %d rocksdb batches, %d log entries)",
-		snapType, snap.Metadata.Index, inSnap.SnapUUID.Short(),
-		size, len(inSnap.Batches), len(inSnap.LogEntries))
+		// Time to clear unreplicated range-ID local keys and update unreplicated
+		// state.
+		unreplicatedState time.Time
+		// Time to process subsumed replicas.
+		subsumedReplicas time.Time
+		// Time to ingest SSTs.
+		ingestion time.Time
+	}
+	log.Infof(ctx, "applying %s snapshot [id=%s index=%d]",
+		snapType, inSnap.SnapUUID.Short(), snap.Metadata.Index)
 	defer func(start time.Time) {
 		now := timeutil.Now()
-		log.Infof(ctx, "applied %s snapshot in %0.0fms [clear=%0.0fms batch=%0.0fms entries=%0.0fms commit=%0.0fms]",
-			snapType, now.Sub(start).Seconds()*1000,
-			stats.clear.Sub(start).Seconds()*1000,
-			stats.batch.Sub(stats.clear).Seconds()*1000,
-			stats.entries.Sub(stats.batch).Seconds()*1000,
-			stats.commit.Sub(stats.entries).Seconds()*1000)
-	}(timeutil.Now())
-
-	// Use a more efficient write-only batch because we don't need to do any
-	// reads from the batch.
-	batch := r.store.Engine().NewWriteOnlyBatch()
-	defer batch.Close()
-
-	// If we're subsuming a replica below, we don't have its last NextReplicaID,
-	// nor can we obtain it. That's OK: we can just be conservative and use the
-	// maximum possible replica ID. preDestroyRaftMuLocked will write a replica
-	// tombstone using this maximum possible replica ID, which would normally be
-	// problematic, as it would prevent this store from ever having a new replica
-	// of the removed range. In this case, however, it's copacetic, as subsumed
-	// ranges _can't_ have new replicas.
-	const subsumedNextReplicaID = math.MaxInt32
-
-	// As part of applying the snapshot, we may need to subsume replicas that have
-	// been merged into this range. Destroy their data in the same batch in which
-	// we apply the snapshot.
-	for _, sr := range subsumedRepls {
-		if err := sr.preDestroyRaftMuLocked(
-			ctx, r.store.Engine(), batch, subsumedNextReplicaID, true, /* destroyData */
-		); err != nil {
-			return err
+		totalLog := fmt.Sprintf(
+			"total=%0.0fms ",
+			now.Sub(start).Seconds()*1000,
+		)
+		unreplicatedStateLog := fmt.Sprintf(
+			"unreplicatedState=%0.0fms ",
+			stats.unreplicatedState.Sub(start).Seconds()*1000,
```

Comment:
I thought we didn't want to print this unless the legacy state is present (i.e. never). I'm also OK nuking this unconditionally since it's such a small write and won't ever matter